packages/performance/src/utils/attributes_utils.ts

```diff
@@ -29,7 +29,6 @@ export enum VisibilityState {
   UNKNOWN = 0,
   VISIBLE = 1,
   HIDDEN = 2,
-  PRERENDER = 3,
```

Comment:
Can you please add a comment here so it is not weird that the enum jumps from 2 to 4?

Reply:
Nvm, we can remove unloaded as well.

Reply:
Removed `UNLOADED`